### ISSUE TITLE
CreateFile not call when file name contains colon?

### ISSUE THREAD

**Issue by barcelona-messi** [keybase/kbfs#1752]
On Windows, CreateFile can call when file name contains *?<>|", but CreateFile not call when file name contains colon, why?

**Comment by strib** [comment]
Windows doesn't allow colons in filenames: https://www.quora.com/Why-are-colons-not-allowed-in-the-filenames-of-Windows-files

**Comment by barcelona-messi** [comment]
@strib ,yes, Windows doesn't allow colons in filenames, but dokan allow colons in filename, ZwCreateFile function can callback when i use dokan c lib, ZwCreateFile function can not callback when i use kbfs

**Comment by strib** [comment]
I assume we don't allow it so as not to confuse certain Windows applications.  @taruti might remember more.

**Comment by taruti** [comment]
Allowing files that won't work with common applications could lead to problems. Especially in shared folders with multiple people.

@barcelona-messi what kind of use-cases do you have? E.g. for maildirs usually an alternate separator character is used on Windows based systems.

**Comment by barcelona-messi** [comment]
@taruti , file name allow contain colons in iOS or linux, i want mount iOS  or linux filesystem
